### PR TITLE
Update email verification connector to use new endpoint

### PR DESCRIPTION
**Pull request by Jxr227**

### CODE REVIEW

File: app/connectors/EmailVerificationConnector.scala [hunk 1]
```diff
@@ -33,15 +33,15 @@ import scala.concurrent.Future
 class EmailVerificationConnector @Inject()(http: HttpClient,
                                            appConfig: AppConfig) {
 
-  private[connectors] def checkVerifiedEmailUrl(email: String): String =
-    s"${appConfig.emailVerificationBaseUrl}/email-verification/verified-email-addresses/$email"
+  private val checkVerifiedEmailUrl: String =
```
Comment (davidmthornton) [inline app/connectors/EmailVerificationConnector.scala]: Does this need to be left as def so we have the flexibility of being able to change the url in config without having to redeploy the app?

File: app/connectors/EmailVerificationConnector.scala [hunk 2]
```diff
@@ -33,15 +33,15 @@ import scala.concurrent.Future
...
+    s"${appConfig.emailVerificationBaseUrl}/email-verification/verified-email-check"
 
   private[connectors] lazy val createEmailVerificationRequestUrl: String =
```
Comment (davidmthornton) [inline app/connectors/EmailVerificationConnector.scala]: Wondering if this needs to be def for the same reason?

File: test/connectors/EmailVerificationConnectorSpec.scala [hunk 3]
```diff
@@ -24,12 +24,6 @@ class EmailVerificationConnectorSpec extends ControllerBaseSpec {
   lazy val connector = new EmailVerificationConnector(mock[HttpClient], mockConfig)
 
   "EmailVerificationConnector" should {
-
-    "generate the correct verify check URL" in {
```
Comment (davidmthornton) [inline test/connectors/EmailVerificationConnectorSpec.scala]: Do you still need to have a test for this function for coverage?

File: app/connectors/EmailVerificationConnector.scala (hunk 1 above)
Comment (Jxr227) [reply app/connectors/EmailVerificationConnector.scala]: App-config changes always require a redeploy to take affect, the app-config is downloaded and bundled with the release, isn't it?

File: test/connectors/EmailVerificationConnectorSpec.scala (hunk 3 above)
Comment (Jxr227) [reply test/connectors/EmailVerificationConnectorSpec.scala]: That's a good point i'll remove it in both

**Comment by Jxr227** [comment]
vat-correspondence-details-frontend PR was open first 👍

File: app/connectors/EmailVerificationConnector.scala (hunk 1 above)
Comment (davidmthornton) [reply app/connectors/EmailVerificationConnector.scala]: Thanks that's good to know :)

**Comment by davidmthornton** [comment]
This looks good James - Approved.

**Pull request merged**